modules/docs/mdx/13.0-UPGRADE-GUIDE.mdx

```diff
@@ -219,44 +281,43 @@ import {dubLogoBlue} from "@workday/canvas-kit-react/common"
 import {dubLogoPrimary} from "@workday/canvas-kit-preview-react/common"
 
 <div dangerouslySetInnerHTML={__html: dubLogoPrimary} />
-
-// In v13
+```
+**After in v13**
+```tsx
 // New logos have been promoted to Main common AND renamed
 import {dubLogoPrimary} from "@workday/canvas-kit-preview-react/common"
 
 <div dangerouslySetInnerHTML={__html: dubLogoPrimary} />
 ```
 
-🤖 The codemod will handle the change of `'dubLogoBlue', 'dubLogoWhite', 'wdayLogoBlue', 'wdayLogoWhite'` to
-`'dubLogoPrimary', 'dubLogoReversed', 'wdayLogoPrimary', 'wdayLogoReversed'` as shown above in the table. It will also handle
-updating your imports.
-
-From:
-```tsx
-import {dubLogoBlue} from "@workday/canvas-kit-react/common"
-```
-To:
-```tsx
-import {dubLogoPrimary} from "@workday/canvas-kit-react/common"
-```
+> 🤖 The codemod will handle the change of `'dubLogoBlue', 'dubLogoWhite', 'wdayLogoBlue', 'wdayLogoWhite'` to
```

Comment:
Question: Do all logo import names get updated? Can we say "all are updated" instead?

Reply:
Yeah good call out, updated.